test/unit/processing/account-codes/select-line-code.test.js

```diff
@@ -0,0 +1,34 @@
+const { selectLineCode } = require('../../../../app/processing/account-codes/select-line-code')
```

Comment:
I think this should be below the constants

Reply:
Agreed.  Co-pilot is good at putting them under the last constant which isn't what I always want!!